### ISSUE TITLE
Remove tweets with videos from the dataset

### ISSUE THREAD

**Issue by ShaoDonCui** [uestc-lj/CFFN#2]
I noticed that the paper mentioned that tweets with videos were removed from the dataset. But I didn't find any tweets with videos, can you help me?
After data processing, the Twitter dataset training set has 14768 members and the test set has 2584 members. This is inconsistent with what was mentioned in previous studies.
